mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -183,6 +183,19 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext
       case Row(features: Vector, expected: Vector) =>
         assert(features ~== expected absTol 1e-14)
     }
+
+    // CountVectorizer test
```

Comment:
Might want to write a better comment here and on ln178 to indicate that `cv` is testing binary toggle creating model from vocab and `cv2` is testing from fitting the model to a df